dir.props

```diff
@@ -99,6 +99,35 @@
 
   <Import Project="$(MSBuildThisFileDirectory)pkg/ExternalPackages/versions.props" Condition="Exists('$(MSBuildThisFileDirectory)pkg/ExternalPackages/versions.props')" />
 
+  <!-- Dependency auto-upgrade configuration -->
+  <ItemGroup>
+    <DependencyBuildInfo Include="CoreFx">
+      <RawUrl>https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master</RawUrl>
```

Comment:
Hmm, yeah, that wouldn't help as much with dev/api (since that branch doesn't exist across all repos) but in that case it can just be unparameterized a little.